src/random.cpp

```diff
@@ -107,36 +194,36 @@ static void InitHardwareRand() {}
 static void ReportHardwareRand() {}
 #endif
 
-static bool GetHardwareRand(unsigned char* ent32) noexcept {
+static bool SeedHardwareFast(CSHA512& hasher) noexcept {
```

Comment:
Returns `bool` that no one reacts to. Drop or log?

Reply:
Same here: if returning `bool` then please consider `NODISCARD`.